### PR TITLE
Closes #151

### LINKED ISSUE #151: Create dataset loader for BioNLP-ST 2011 REL
- **Name:** BioNLP-ST 2011 REL
- **Description:** Part-of relations between genes and other biological entities
- **Task:** RE
- **Paper:** https://aclanthology.org/W11-1812.pdf
- **Data:** [https://github.com/openbiocorpora/bionlp-st-2011-rel]()
- **License:** License Terms for BioNLP Shared Task 2011 Data

### PR DESCRIPTION
**Pull request by barthfab**
### Checkbox

- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `biodatasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_BIGBIO_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `BigBioConfig` for the source schema and one for a bigbio schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_bigbio biodatasets/my_dataset/my_dataset.py`.
- [x] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

**Review by hakunanatasha — CHANGES_REQUESTED** [review]
Hi @barthfab 

I'm getting this error when I run the unittests:

```
FAIL: runTest (__main__.TestDataLoader) [Check schema validity]
Run all tests that check:
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/natasha/Projects/hfbiomed/biomedical/tests/test_bigbio.py", line 115, in runTest
    self.test_schema(schema)
  File "/home/natasha/Projects/hfbiomed/biomedical/tests/test_bigbio.py", line 544, in test_schema
    raise AssertionError(f"Required key '{non_empty_feature}' does not have any instances")
AssertionError: Required key 'entities' does not have any instances

----------------------------------------------------------------------
Ran 1 test in 6.508s

FAILED (failures=1)
```

I run the unit-tests with:
`python -m tests.test_bigbio biodatasets/bionlp_st_2011_rel/bionlp_st_2011_rel.py` after I place your file there. Can you let me know if you can fix this? Happy to merge once they pass. If you need help digging through the file, I'm happy to assist.

File: biodatasets/bionlp_st_2011_rel/bionlp_st_2011_rel.py [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+_HOMEPAGE = "https://github.com/openbiocorpora/bionlp-st-2011-rel"
+
+_LICENSE = "License Terms for BioNLP Shared Task 2011 Data"
+
+_URLs = {
+    "source": "https://github.com/openbiocorpora/bionlp-st-2011-rel/archive/refs/heads/master.zip",
+    "bigbio_kb": "https://github.com/openbiocorpora/bionlp-st-2011-rel/archive/refs/heads/master.zip",
+}
+
+_SUPPORTED_TASKS = [Tasks.RELATION_EXTRACTION]
```
Comment (leonweber) [inline biodatasets/bionlp_st_2011_rel/bionlp_st_2011_rel.py:70]: This should include NER as well

**Review by leonweber — CHANGES_REQUESTED** [review]
Looks good to me. Failure for test_schema is expected because in all BioNLP shared task datasets test set annotations are hidden and thus missing from our version.

**Review by leonweber — APPROVED** [review]
Looks good to me. Failure for test_schema is expected because in all BioNLP shared task datasets test set annotations are hidden and thus missing from our version.

**Pull request merged**